### PR TITLE
590 dw fillable pdf issue after metadata stripping update

### PR DESCRIPTION
**Pull request by alphaomega325**
Changed some commands and removed ghostscript due to 2 pass of qpdf and exiftools removed all unnecessary metadata and ghostscript doesn't help in that regard.  Also added in mutools to allow for optimization of pdfs.

### CODE REVIEW

**Comment by alphaomega325** [comment]
Solved the flake8 issue.

File: castle/cms/commands.py [hunk 1]
```diff
@@ -309,7 +345,28 @@ def convert_to_pdf(self, filepath, filename, output_dir):
...
+    loffice = LibreOfficeSubProcess()
 except IOError:
-    logger.exception("No docsplit installed.")
-    docsplit = None
+    logger.exception("Libreoffice not installed. castle.cms"
+                     "will not be able to convert text files to pdf.")
+    loffice = None
+
+
+class DocSplitSubProcess(BaseSubProcess):
```
Comment (lucid-0) [inline castle/cms/commands.py]: is this still supposed to be called DocSplit subprocess? Could be moved somewhere now that we won't be using docsplit.

File: castle/cms/interfaces/cache.py~ [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from zope import schema
```
Comment (lucid-0) [inline castle/cms/interfaces/cache.py~]: I see your addition of *~ to .gitignore was that intended to exclude this file?

File: travis.cfg [hunk 3]
```diff
@@ -32,6 +32,7 @@ eggs +=
     castle.cms[test]
     celery[sqlalchemy]
     collective.celery
+    collective.documentviewer
```
Comment (lucid-0) [inline travis.cfg]: we don't want to commit this change, as documentviewer will still generally be installed as it is currently.

File: castle/cms/interfaces/cache.py~ (hunk 2 above)
Comment (alphaomega325) [reply castle/cms/interfaces/cache.py~]: Yes it was suppose to ignore all backup files that emacs automatically create.

File: castle/cms/commands.py (hunk 1 above)
Comment (alphaomega325) [reply castle/cms/commands.py]: I did this to provide backwards compatibility with any possible addons that have been added on by our users.

Comment (alphaomega325) [reply castle/cms/commands.py]: Or to be more accurately, users that use castle.cms but doesn't use our services.

File: travis.cfg (hunk 3 above)
Comment (alphaomega325) [reply travis.cfg]: Just removed it.

File: castle/cms/interfaces/cache.py~ (hunk 2 above)
Comment (alphaomega325) [reply castle/cms/interfaces/cache.py~]: And removed.

**Pull request closed without merging**